### DISCUSSION TITLE
UX: strip preferred file extension?

### DISCUSSION THREAD

**Discussion by seankhliao** [FiloSottile/age#246] [UX reports]
## What were you trying to do
decrypt a set of files with `find`

## What happened
I need to invoke a shell for every file I wish to decrypt just to strip `.age` from the filename (passed with `{}`)

**Comment by FiloSottile** [comment]
The behavior of `age` is not to put the output in a file by default, like say `gunzip` does, so there is no place to strip the extension automatically.
